cub/cub/util_macro.cuh

```diff
@@ -49,19 +49,8 @@
 
 CUB_NAMESPACE_BEGIN
 
-#ifndef CUB_ALIGN
-#  if defined(_WIN32) || defined(_WIN64)
-/// Align struct
-#    define CUB_ALIGN(bytes) __declspec(align(32))
```

Comment:
o.O hardcoded 32

Reply:
OMG :D Thankfully, this was not used anywhere.